models/public/erfnet/accuracy-check.yml

```diff
@@ -0,0 +1,70 @@
+models:
+  - name: ERFNet
+    launchers:
+      - framework: onnx_runtime
+        device: CPU
+        model: erfnet.onnx
+        adapter:
+          type: segmentation
+          make_argmax: True
+    datasets:
+      - name: erfnet_data
+        preprocessing:
+          - type: resize
+            dst_width: 976
+            dst_height: 208 
+        postprocessing:
+          - type: resize_segmentation_mask
+            dst_width: 1640
+            dst_height: 350
+            apply_to: prediction
+        annotation_conversion:
+          converter: voc_segmentation
+          imageset_file: val.txt
+          images_dir: JPEGImages/
+          mask_dir: Annotations/
+          dataset_meta_file: erfnet_meta_zxw.json
+        data_source: ./JPEGImages/
+        additional_data_source: ./Annotations/
+        metrics:
+          - type: segmentation_accuracy
+            use_argmax: false
+            presenter:  print_scalar
+          - type: mean_iou
+            use_argmax: false
+            presenter:  print_scalar
+  - name: ERFNet
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        model: erfnet.xml
+        weights: erfnet.bin
```

Comment:
```suggestion

```
all OMZ accuracy configs should not contain device and path to model for ability to test model on different devices and in different precisions